### PR TITLE
[FL-3426] "More" button on "Info" scene shows card dump for Ultralight cards

### PR DESCRIPTION
**Pull request by RebornedBrain**
# What's new

- [ Describe changes here ]

# Verification 

- [ Describe how to verify changes ]

# Checklist (For Reviewer)

- [ ] PR has description of feature/bug or link to Confluence/Jira task
- [ ] Description contains actions to verify feature/bugfix
- [ ] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: applications/main/nfc/scenes/nfc_scene_card_dump.c [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+void nfc_scene_card_dump_on_enter(void* context) {
+    nfc_protocol_support_on_enter(NfcProtocolSupportSceneCardDump, context);
+}
+
+bool nfc_scene_card_dump_on_event(void* context, SceneManagerEvent event) {
+    UNUSED(context);
+    UNUSED(event);
+    return false;
+    //return nfc_protocol_support_on_event(NfcProtocolSupportSceneCardDump, context, event);
+}
```
Comment (gornekich) [inline applications/main/nfc/scenes/nfc_scene_card_dump.c]: it's better to implement `nfc_protocol_support_on_event()` function and call it from here. Yes, the implementation is trivial, but it's better to place all logic in one place - nfc_protocol_support.c in our case

File: applications/main/nfc/helpers/protocol_support/nfc_protocol_support.c [hunk 2]
```diff
@@ -489,6 +500,10 @@ static const NfcProtocolSupportCommonSceneBase
                 .on_event = nfc_protocol_support_scene_info_on_event,
                 .on_exit = nfc_protocol_support_scene_info_on_exit,
             },
+        [NfcProtocolSupportSceneCardDump] =
+            {.on_enter = nfc_protocol_support_scene_card_dump_on_enter,
+             .on_event = NULL,
+             .on_exit = nfc_protocol_support_scene_card_dump_on_exit},
```
Comment (gornekich) [inline applications/main/nfc/helpers/protocol_support/nfc_protocol_support.c]: Please, add ',' after `nfc_protocol_support_scene_card_dump_on_exit` and run linter: ./fbt format

File: applications/main/nfc/helpers/protocol_support/mf_ultralight/mf_ultralight.c [hunk 3]
```diff
@@ -185,6 +197,7 @@ const NfcProtocolSupportBase nfc_protocol_support_mf_ultralight = {
             .on_enter = nfc_scene_info_on_enter_mf_ultralight,
             .on_event = nfc_scene_info_on_event_mf_ultralight,
         },
+    .scene_card_dump = {.on_enter = nfc_scene_card_dump_on_enter_mf_ultralight, .on_event = NULL},
```
Comment (gornekich) [inline applications/main/nfc/helpers/protocol_support/mf_ultralight/mf_ultralight.c]: Add common: `.on_event = NULL,` and run linter

**Review by gornekich — CHANGES_REQUESTED** [review]
Good job! Please fix issues I mentioned

File: applications/main/nfc/scenes/nfc_scene_card_dump.c (hunk 1 above)
Comment (RebornedBrain) [reply applications/main/nfc/scenes/nfc_scene_card_dump.c]: Done

File: applications/main/nfc/helpers/protocol_support/nfc_protocol_support.c (hunk 2 above)
Comment (RebornedBrain) [reply applications/main/nfc/helpers/protocol_support/nfc_protocol_support.c]: Done

File: applications/main/nfc/helpers/protocol_support/mf_ultralight/mf_ultralight.c (hunk 3 above)
Comment (RebornedBrain) [reply applications/main/nfc/helpers/protocol_support/mf_ultralight/mf_ultralight.c]: Done

**Review by gornekich — APPROVED** [review]

**Pull request merged**